vcpkg.json

```diff
@@ -1,20 +1,31 @@
 {
-    "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
-    "name": "dosbox-staging",
-    "description": "DOSBox Staging is a modern continuation of DOSBox with advanced features and current development practices.",
-    "version": "0.82.0-alpha",
-    "dependencies": [
-        "fluidsynth",
-        "gtest",
-        "iir1",
-        "libmt32emu",
-        "libpng",
-        "opusfile",
-        "sdl2",
-        "sdl2-image",
-        "sdl2-net",
-        "speexdsp",
-        "tracy",
-        "zlib"
-    ]
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
+  "name": "dosbox-staging",
+  "description": "DOSBox Staging is a modern continuation of DOSBox with advanced features and current development practices.",
+  "version": "0.81.0-alpha",
+  "dependencies": [
+    "fluidsynth",
+    "gtest",
+    "iir1",
+    "libmt32emu",
+    "libpng",
+    "libslirp",
+    "opusfile",
+    "sdl2",
+    "sdl2-image",
+    "sdl2-net",
+    "speexdsp",
+    "tracy",
+    "zlib"
```

Comment:
Is zlib-ng available in vcpkg?  On Linux, we are using zlib-ng for release builds and system zlib for debug builds.  There is a significant performance advantage for recording video with zmbv.

Reply:
Yep, we absolutely want to keep zlib-ng.